package.json

```diff
@@ -19,6 +19,7 @@
     "generate:iife": "rollup -c -f iife",
     "hack:iife": "npm run copy:iife && npm run rename:iife && npm run package:iife",
     "copy:iife": "copyfiles -u 2 src/client-modules/docs.iife.min.js node_modules/iife",
+    "copy:dsconfig": "rimraf --glob ./static/dsconfig/* && copyfiles -u 4 node_modules/@genesislcap/design-system-configurator/dist/* static/dsconfig",
```

Comment:
Why not just `rimraf static/dsconfig`?

Reply:
the (*) asterik was causing a fail and giving me an error but adding --glob seemed to fix it. I did think it would enable globbing by default but with the latest version it was still giving an error

Reply:
I meant without asterisk - deleting the whole folder. Copyfiles creates target dir if it doesn't exist